pkgs/top-level/python-packages.nix

```diff
@@ -9165,6 +9165,32 @@ in modules // {
     };
   };
 
+  shinken = buildPythonPackage rec {
+    version = "2.4.3";
+    name = "Shinken-${version}";
+    disabled = isPy3k;
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/S/Shinken/${name}.tar.gz";
+      sha256 = "1hxbnxlk2bacb7p6lim9kqy50581m0d3p3y6b9hw0x800lfk0y6q";
+    };
+
+    dontStrip = true;
+
+    # missing: logging tagging
+
+    propagatedBuildInputs = with self; [
+      arrow bottle cffi cherrypy django paramiko pycurl pymongo pyopenssl
+    ];
+
+    meta = {
+      homepage = http://www.shinken-monitoring.org/;
+      description = "Shinken Monitoring Framework";
```

Comment:
PyPI pages of packages often have a nice one-liner at the top. In this case *Shinken is a monitoring framework compatible with Nagios configuration and plugins*